modsecurity/cppbuild.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# This file is meant to be included by the parent cppbuild.sh script
+if [[ -z "$PLATFORM" ]]; then
+    pushd ..
+    bash cppbuild.sh "$@" modsecurity
+    popd
+    exit
+fi
+
+mkdir -p $PLATFORM
+cd $PLATFORM
+INSTALL_PATH=`pwd`
+
+if   [ ! -d "ModSecurity" ];
+then
+  git clone https://github.com/SpiderLabs/ModSecurity
+  cd ModSecurity
+  git checkout origin/v3/master
+  git submodule init
+  git submodule update
```

Comment:
They seem to offer proper source code bundles for releases:
https://github.com/SpiderLabs/ModSecurity/releases
Say we replace these `git` calls with something like the following, does it still work?
```
download https://github.com/SpiderLabs/ModSecurity/releases/download/v3.0.4/modsecurity-v3.0.4.tar.gz modsecurity-v3.0.4.tar.gz
```

Reply:
I need to check it. I will answer as soon as check it.

Reply:
I checked it. Unfortunately, it not works. There were a lot of changes from the previous release and it seems that release rules will be changed. I think it will work after the release 3.1.0.  And also their current recommendation to build from the master branch,  maybe it associated with the release policy.

Reply:
Hum, ok, let's wait and see for 3.1.0? How soon is that going to happen?

Reply:
We could also set the version of the presets to `master`, temporarily, until 3.1.0 gets released. What do you think?

Reply:
Yep, we can set the version of the preset to master, temporarily, and then change it to 3.0.5 after the release. "Migration from master and 3.0.5 to 3.1.0 will be much easier" - it`s a [comment](https://github.com/SpiderLabs/ModSecurity/issues/2534) which I got on the issue about next release. Thanks a lot :+1: